tests/api/test_additional_admin_api.py

```diff
@@ -0,0 +1,353 @@
+import logging
+from unittest.mock import AsyncMock
+
+from httpx import AsyncClient, HTTPStatusError, Request, Response
+from pytest_mock import MockerFixture
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app import Settings
+from app.api_clients.optscale import UserDoesNotExist
+from app.db.models import Organization
+from app.enums import OrganizationStatus
+from tests.types import ModelFactory
+
+
+async def test_add_additional_admin_request(
+    mocker: MockerFixture,
+    get_existing_user,
+    create_org_employee_status_409,
+    operations_client: AsyncClient,
+    db_session: AsyncSession,
+    organization_factory: ModelFactory[Organization],
+    caplog,
+    test_settings: Settings,
+    httpx_mock,
+):
+    organization = await organization_factory(
+        name="ACME CO",
+        currency="EUR",
+        operations_external_id="initial_external_id",
+    )
+    mocker.patch(
```

Comment:
In all the other tests, we used pytest-httpx to mock api calls to 3rd party APIs. Please use it